src/yarpmanager/src-manager/logwidget.cpp

```diff
@@ -3,7 +3,8 @@
 #include <QCoreApplication>
 
 LogWidget::LogWidget(QWidget *parent) :
-    QListWidget(parent)
+    QListWidget(parent),
+    contextMenu(nullptr)
```

Comment:
In `master` you have to put `YARP_NULLPTR`

Reply:
Here using `YARP_NULLPTR` was giving an 'undefined symbol' error

Reply:
Ok I included `compiler.h` to avoid using `nullptr` https://github.com/robotology/yarp/pull/1467/commits/6ae143d8afb84690ad94174b1de1ee659eae4fc6